src/relay/ir/indexed_graph.h

```diff
@@ -19,7 +19,11 @@
 
 /*!
  * \file src/relay/ir/indexed_graph.h
- * \brief A pattern matcher for matching dataflow properties.
+ * \brief A graph representation of the dataflow in a Relay expression or Relay dataflow
+ * pattern. Nodes in this graph capture generic dataflow inputs, outputs, dominators and control
+ * flow scopes.
+ *
+ * TODO(mbs): Rename to 'DataflowGraph'.
```

Comment:
I think either we should do the renaming, or remove the TODO.

Reply:
ack, removed.